data/governments.txt

```diff
@@ -505,6 +506,18 @@ government "Hai (Unfettered)"
 	"hostile hail" "hostile unfettered"
 	"hostile disabled hail" "hostile disabled unfettered"
 
+government "Hai (Unfettered) (Wanderer Tribute)"
```

Comment:
I'd rename this to smth like: "Hai (Unfettered Wanderer Tribute)"
The double parenthesis feels weird